csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -543,6 +543,20 @@ public override string ToString()
             }
         }
 
+#if NETSTANDARD2_0
+        /// <summary>
+        /// Creates a new span over the RepeatedField.  This is the most
+        /// efficient way to read and write values to the RepeatedField because
+        /// there are no internal null or bounds checks.  You are responsible
+        /// for ensuring no null values are written to the span.

```

Comment:
I think that the lack of null check is dangerous and relying on users to always to the right thing is dangerous.

Reply:
C++ allows you to memcpy into RepeatedField.  But that's C++.  I suppose C# users expect the null checks and such.